SConstruct

```diff
@@ -39,6 +43,11 @@ env = Environment(
     "-O2",
     "-Wunused",
     "-Werror",
+    "-Werror=implicit-function-declaration",
```

Comment:
If you want to enable these, let's do it in a separate PR. also, this should match openpilot